src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -28,9 +26,7 @@ if [%BUILDTOOLS_USE_CSPROJ%]==[] (
 { ^
   "dependencies": ^
     { ^
-      "MicroBuild.Core": "%MICROBUILD_VERSION%", ^
-      "Microsoft.Portable.Targets": "%PORTABLETARGETS_VERSION%", ^
-      "Microsoft.Net.Compilers": "%ROSLYNCOMPILERS_VERSION%" ^
+      "MicroBuild.Core": "%MICROBUILD_VERSION%" ^
```

Comment:
I don't need it to build myself, but I thought this was required for official builds which is why I didn't touch that.